### PR TITLE
[ISV-1344] As a partner, the hosted pipeline won't re-test my operator if only project settings are changed

### PR DESCRIPTION
**Pull request by haripate**

### CODE REVIEW

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml [hunk 1]
```diff
@@ -71,7 +71,8 @@ spec:
 
         if [ -f $RESULTS_FILE ]; then
           echo "Results exist"
-          echo -n "true" | tee $(results.preflight_results_exists.path)
+          CHECK_RESULTS=$(cat $RESULTS_FILE | jq -r '.passed')
```
Comment (amisstea) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml:74]: Please add a log message to tell the user if these results passed.

Comment (haripate) [reply ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml:74]: Updated!

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml [hunk 2]
```diff
@@ -72,6 +72,10 @@ spec:
         if [ -f $RESULTS_FILE ]; then
           echo "Results exist"
           CHECK_RESULTS=$(cat $RESULTS_FILE | jq -r '.passed')
+          if [ $CHECK_RESULTS = "false" ]; then
+            echo "Not all preflight tests passed."
+          fi
+          echo "All preflight tests passed successfully."
```
Comment (amisstea) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml:78]: Shouldn't this be inside an else condition?

Comment (haripate) [reply ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml:78]: Doesn't this work as else condition?

**Review by amisstea — APPROVED** [review]

**Pull request merged**